hbase-server/src/main/java/org/apache/hadoop/hbase/wal/BoundedRecoveredEditsOutputSink.java

```diff
@@ -83,25 +85,26 @@ public void append(EntryBuffers.RegionEntryBuffer buffer) throws IOException {
   }
 
   @Override
-  public List<Path> close() throws IOException {
+  public List<Path> close(MonitoredTask status) throws IOException {
```

Comment:
Does this context have access to the OutputSink's MonitoredTask field? Can we do that instead, so we don't have to change these method signatures?